app/src/main/java/com/grzegorzdec/tagmango/tools/RegistryViewModel.kt

```diff
@@ -0,0 +1,31 @@
+package com.grzegorzdec.tagmango.tools
+
+import androidx.databinding.Observable
+import androidx.databinding.PropertyChangeRegistry
+import androidx.lifecycle.ViewModel
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.cancel
+import kotlin.coroutines.CoroutineContext
+
+open class RegistryViewModel: ViewModel(), Observable {
+
+    protected val registry = PropertyChangeRegistry()
```

Comment:
if we wanna keep this class for now, make it private and add `fun addOnpropertyChangeListener`

Reply:
This is copied from the last version that worked, why do we need this method?

Reply:
I mean `fun onPropertyChange`, thanks to that you can call `onPropertyChange(BR.something)` instead of `registry.onPropertyChange(BR.something)` but leave it for now, I'm working on rx decorator